api/src/main/java/edu/cornell/mannlib/vitro/webapp/controller/edit/FauxPropertyRetryController.java

```diff
@@ -72,14 +72,24 @@ public void doPost(HttpServletRequest req, HttpServletResponse response) {
 		req.setAttribute("scripts", "/templates/edit/formBasic.js");
 		req.setAttribute("title", "Faux Property Editing Form");
 		req.setAttribute("_action", epo.getAction());
+		AccessObjectType aot;
+		if (populator.isFauxDataProperty()) {
+		    aot = AccessObjectType.FAUX_DATA_PROPERTY;
+		} else {
+		    aot = AccessObjectType.FAUX_OBJECT_PROPERTY;
+		}
+        req.setAttribute("_faux_property_type", aot);
```

Comment:
please fix the indentation of this line.

Reply:
fixed